### PR TITLE
Save rendezvous IP in a file 'rendezvousIP' alongside the ISO

### PR DESCRIPTION
**Pull request by pawanpinjarkar**
Signed-off-by: Pawan Pinjarkar <ppinjark@redhat.com>

### CODE REVIEW

File: pkg/asset/agent/image/ignition.go [hunk 1]
```diff
@@ -120,8 +121,15 @@ func (a *Ignition) Generate(dependencies asset.Parents) error {
 	if err != nil {
 		return err
 	}
+
 	logrus.Infof("The rendezvous host IP (node0 IP) is %s", nodeZeroIP)
 
+	err = ioutil.WriteFile(filepath.Join(".", "rendezvousIP"), []byte(nodeZeroIP), 0)
```
[the remark this answers is not in the document]
Comment (bfournie) [reply pkg/asset/agent/image/ignition.go:127]: It looks like this is being saved where the command is run and not with the ISO when I tested it out:

```
[stack@host06 dev-scripts-latest]$ pwd
/home/stack/dev-scripts-latest
[stack@host06 dev-scripts-latest]$ ls rendezvousIP 
rendezvousIP
[stack@host06 dev-scripts-latest]$ ls ocp/ostest/agent.x86_64.iso 
ocp/ostest/agent.x86_64.iso
[stack@host06 dev-scripts-latest]$ 



```

[the remark this answers is not in the document]
Comment (bfournie) [reply pkg/asset/agent/image/ignition.go:127]: Also maybe need to set permissions
```
[stack@host06 dev-scripts-latest]$ cat rendezvousIP 
cat: rendezvousIP: Permission denied
[stack@host06 dev-scripts-latest]$ sudo cat rendezvousIP 
192.168.111.80[stack@host06 dev-scripts-latest]$
```

[the remark this answers is not in the document]
Comment (zaneb) [reply pkg/asset/agent/image/ignition.go:127]: I think we need to do this inside `AgentImage.PersistToFile()` to get it in the correct directory.

[the remark this answers is not in the document]
Comment (pawanpinjarkar) [reply pkg/asset/agent/image/ignition.go]: yes

**Comment by bfournie** [comment]
/lgtm

Tested in dev-scripts and looks good

**Comment by bfournie** [comment]
Is there anything preventing this from being cherry-picked to 4.12?

**Pull request merged**